### DISCUSSION TITLE
Installation failed - Error: Connection activation failed: Unknown error

### DISCUSSION THREAD

**Discussion by talibosmani** [Security-Onion-Solutions/securityonion#13541] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

8.443875

### Storage for /nsm

14.525117

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I am installing on XCP-NG, the installation works fine in the first stage, the machine reboot as expected. Then in the second state of installations after almost an hour of installation i see it fails. 

![2024-08-22 22_04_54-admin@seconion_~](https://github.com/user-attachments/assets/e237fe39-e4c8-4742-928f-d67ac789eb7d)
![2024-08-22 22_01_27-admin@seconion_~](https://github.com/user-attachments/assets/49edf6e9-6ab3-4870-8c75-b113b54c93e7)
![2024-08-22 22_00_43-admin@seconion_~](https://github.com
[errors.log](https://github.com/user-attachments/files/16721109/errors.log)
/user-attachments/assets/2d6f0f52-81f4-435e-9e29-1cc685833a25)
[sosetup.log](https://github.com/user-attachments/files/16721108/sosetup.log)
![2024-08-22 22_00_43-admin@seconion_~](https://github.com/user-attachments/assets/66f4b86f-cdf7-40cb-99a1-f1fb6e1ef47b)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
What happens if you run` sudo so-checkin`? Does it complete without error? Also, what does `sudo nmcli con show` return?

**Reply by talibosmani** [reply]
Hi, 

Please see the screenshot below.


![image](https://github.com/user-attachments/assets/ddfec114-88b2-471c-80e2-d192d57ab087)

While running the so-checkin no error:

Summary for local
--------------
Succeeded: 852 (changed=35)
Failed:      0
--------------
Total states run:     852
Total run time:    65.575 s
[admin@seconion ~]$

**Reply by cm-ops** [reply]
Did you run this from the sosetup.log?` journalctl -xe NM_CONNECTION=8c1e2cf7-87df-42a0-af0d-8edeb5607245 + NM_DEVICE=enX1' to get more details`

**Comment by mario21ic** [comment]
I have the same issue :/

**Comment by eileon** [comment]
The same for me with 2.4.110-20241010 ISO image released on 2024/10/10
i have two disks
When i run 
sudo salt-call state.highstate
i get in red
local :
Data failed to compile
The function "state.highstate" is running as PID - and was started at _ with jid -

i set / on sdb and /nsm on /sda  (i think this leads to an error)

**Reply by eileon** [reply]
Same issue with only one disk on /dev/sda

**Reply by eileon** [reply]
Finally it works : just waiting for some time.
sudo so-status  --> everything running
sudo so-checkin --> here maybe wait some time and be patient, let the system manage with highstate
sudo salt-call state.highstate --> OK
i can connect to https://@IP
Cool!
